client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -0,0 +1,389 @@
+						'Easy-to-use mobile card reader designed for fast, reliable payments.'
+					),
+				},
+			],
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/product-management.png',
+			title: translate( 'Product management' ),
+			subtitle: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+			items: [
+				{
+					title: translate( 'Unlimited products' ),
+					subtitle: translate( 'Add unlimited products to your store.' ),
+				},
+				{
+					title: translate( 'Offer gift cards' ),
+					subtitle: translate( 'Sell and accept pre-paid, multi-purpose e-gift vouchers.' ),
+				},
+				{
+					title: translate( 'Send back in stock notifications' ),
+					subtitle: translate( 'Notify customers when your products are restocked.' ),
+				},
+				{
+					title: translate( 'Set order limits' ),
+					subtitle: translate( 'Specify min and max allowed product quantities for orders.' ),
+				},
+				{
+					title: translate( 'Sell product bundles' ),
+					subtitle: translate( 'Offer personalized product packages and bulk discounts.' ),
+				},
+				{
+					title: translate( 'Offer customizable product kits' ),
+					subtitle: translate(
+						'Use Composite Products to add product kit building functionality with inventory management.'
+					),
+				},
+				{
+					title: translate( 'Import your products via CSV' ),
+					subtitle: translate( 'Import, merge, and export products using a CSV file.' ),
+				},
+				{
+					title: translate( 'Sell product add-ons' ),
+					subtitle: translate( 'Enable gift wrapping/messages or custom pricing.' ),
+				},
+				{
+					title: translate( 'Unlimited images' ),
+					subtitle: translate( 'Add any number of images to your product variations.' ),
+				},
+				{
+					title: translate( 'Product recommendations' ),
+					subtitle: translate(
+						'Earn more revenue with automated upsell and cross-sell product recommendations.'
+					),
+				},
+			],
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/customization.png',
+			title: translate( 'Themes and customization' ),
+			subtitle: translate( 'Bring your brand to life with a fully customizable storefront.' ),
+			items: [
+				{
+					title: translate( 'Premium themes' ),
+					subtitle: translate(
+						'Tap into a diverse selection of beautifully designed premium themes.'
+					),
+				},
+				{
+					title: translate( 'Block-based templates' ),
+					subtitle: translate(
+						"Take control over your store's layout without touching a line of code."
+					),
+				},
+				{
+					title: translate( 'Cart and checkout optimization' ),
+					subtitle: translate(
+						'Streamline your checkout and boost conversions with the Cart and Checkout blocks.'
+					),
+				},
+			],
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/growth.png',
+			title: translate( 'Marketing and growth' ),
+			subtitle: 'Optimize your store for sales by adding in email and social integrations.',
```

Comment:
Note: We should switch this back to lorem ipsum, wrap the content in `translate()`, or make a note of the untranslated string to make sure we don't miss it later.